StompConsumer.php

```diff
@@ -96,17 +97,23 @@ public function receive(int $timeout = 0): ?Message
     {
         $this->subscribe();
 
-        if (0 === $timeout) {
-            while (true) {
-                if ($message = $this->stomp->readMessageFrame($this->subscriptionId, 100)) {
-                    return $this->convertMessage($message);
+        try {
+            if (0 === $timeout) {
+                while (true) {
+                    if ($message = $this->stomp->readMessageFrame($this->subscriptionId, 100)) {
+                        return $this->convertMessage($message);
+                    }
                 }
             }
-        } else {
-            if ($message = $this->stomp->readMessageFrame($this->subscriptionId, $timeout)) {
-                return $this->convertMessage($message);
+            else {
+                if ($message = $this->stomp->readMessageFrame($this->subscriptionId, $timeout)) {
+                    return $this->convertMessage($message);
+                }
             }
         }
+        catch (ErrorFrameException $e) {
+            throw new \Exception($e->getMessage() . "\n" . $e->getFrame()->getBody());
+        }
```

Comment:
I noticed while debugging that the error returned by the PHP stomp library can end up hiding deeper errors.  I've added this to improve error feedback in those scenarios.

_(My specific situation was that I was not using the correct queue name when running my worker, so I kept getting messages about the connection failing, but not the underlying response from the broker saying "that queue doesn't exist". :sweat_smile: )_